drivers/usb/device/usb_dc_stm32.c

```diff
@@ -318,11 +345,13 @@ static int usb_dc_stm32_init(void)
 #endif
 	usb_dc_stm32_state.pcd.Init.dev_endpoints = DT_USB_NUM_BIDIR_ENDPOINTS;
 	usb_dc_stm32_state.pcd.Init.speed = usb_dc_stm32_get_maximum_speed();
+#ifndef CONFIG_SOC_SERIES_STM32F1X
```

Comment:
Did you check with the latest STM32Cube modules? It seems they added `phy_itface` to `USB_OTG_CfgTypeDef` structure.

Reply:
Ah yes, I missed the update, thanks!